server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -84,6 +86,14 @@ private static boolean clusterUUIDCommitted(Object[] fields) {
         return (boolean) fields[9];
     }
 
+    private static int codecVersion(Object[] fields) {
+        return (int) fields[10];
+    }
+
+    private static String globalMetadataFileName(Object[] fields) {
```

Comment:
we should return the filename only in this method.

and created a separate method for globalMetadataFilePath, which returns the entire path